### ISSUE TITLE
Color contrast on side-bar chevrons is too low

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#302]
### Steps to recreate
Install and run an audit using aXe browser extension.

The chevrons in the side-bar have a contrast of 2.49: 1 which is a ratio that doesn't meet AA or AAA WCAG standard.

**Comment by designertyler** [comment]
Good find @dakahn 

Gray 70 #565656 is dark enough for a 7.34:1 ratio

It definitely helps the thin chevron stand out a little more

<img width="342" alt="screen shot 2018-10-26 at 2 33 24 pm" src="https://user-images.githubusercontent.com/19270502/47588701-a253f600-d92c-11e8-8ab9-db659f519905.png">
<img width="323" alt="screen shot 2018-10-26 at 2 37 00 pm" src="https://user-images.githubusercontent.com/19270502/47588702-a253f600-d92c-11e8-8a0d-a384a8c06cb5.png">
